### PR TITLE
rename groupId `com.consol.citrus` to `org.citrusframework`

### LINKED ISSUE #938: move the groupId
as stated in the title: move the `groupId` from `com.consol.citrus` to `org.citrusframework`. this issue is part of #922.

### PR DESCRIPTION
**Pull request by bbortt**
closes #938.

### CODE REVIEW

**Comment by bbortt** [comment]
hardly believable what a mess an IDE can create once you start renaming things 🤦🏼 I had to start over. worth noticing: after 948a2b711bfabb4050fca848f53a483af9550406 the command `./mvnw -pl :citrus-core -am install` passes again.

**Comment by bbortt** [comment]
@christophd I think the PR is ready! however, the build fails. it looks like `citrus-arquillian` tries to fetch the `SNAPSHOT` versopm from remote instead of localhost repository ([source](https://github.com/citrusframework/citrus/actions/runs/4530397798/jobs/7979276017?pr=939#step:6:67121)?

```
Could not find artifact org.citrusframework.citrus:citrus-core:jar:4.0.0-SNAPSHOT, caused by: Could not find artifact org.citrusframework.citrus:citrus-core:jar:4.0.0-SNAPSHOT
```

File: README.md
Comment (christophd) [inline README.md:1]: The badges will be broken until we release 4.0.0-M1 but I am ok with that

**Review by christophd — COMMENTED** [review]
I wonder if we should just use `org.citrusframework` as a base package instead of `org.citrusframework.citrus` as it is kind of duplication in `citrusframework` and `citrus`. we could save a package hierarchy by removing `citrus`.

So we end up with `org.citrusframework.jms` instead of `org.citrusframework.citrus.jms`

WDYT?

Regarding the failing `citrus-arquillian` module I guess we can just ignore as I am about to remove the module anyway.

**Comment by bbortt** [comment]
> [chore(](https://github.com/citrusframework/citrus/pull/939/commits/9a4171ec8fdcf480a6722a8a6298051fe6dc2e24)https://github.com/citrusframework/citrus/issues/938[): Update file paths in tests to org/citrusframework](https://github.com/citrusframework/citrus/pull/939/commits/9a4171ec8fdcf480a6722a8a6298051fe6dc2e24)

how did i miss those? 😅 again.. IDE renaming. what IDE are you using, if I may ask @christophd? I am an IntelliJ people.

the build problem still remains...

**Comment by christophd** [comment]
@bbortt IntelliJ here, too

File: README.md
Comment (bbortt) [reply README.md:1]: fine with me too :)

**Comment by bbortt** [comment]
whey - green build 🚀

**Comment by christophd** [comment]
Oooo nice! bear with me I will have a final look

**Comment by christophd** [comment]
@bbortt I have squashed the commits a bit so we avoid changing ~5000 files multiple times

I also updated all the URLs used in unit and integration tests to `http://citrusframework.org`

**Comment by bbortt** [comment]
@christophd saw your changes: /lgtm. also thought about the licenses, but kept the change focused. glad you adaptet it already - great collaboration going on here 😍

**Pull request merged**